### ISSUE TITLE
Document variables in titanic_ml dataset

### ISSUE THREAD

**Issue by florianhartig** [TheoreticalEcology/ecodata#22]

**Comment by florianhartig** [comment]
All variables (some missing)

@MaximilianPi - should we also remove the variables bodies and life boat?

**Comment by MaximilianPi** [comment]
Yes, let's remove them, the first submissions achieved easily over 0.9 AUC and the feature engineering / data cleaning didn't have much impact on the performance
